### PR TITLE
Add a lockable quickhook retainer

### PR DESCRIPTION
**Pull request by yoannchaudet**
I like quickhooks but wish they would stay in place better.

This is an attempt at addressing with the addition of a new retainer. It is compatible with all quickhooks (it's the same geometry, just wider).

This extra retainer has an orientation and it must be aligned with a panel hole.

To secure the retainer to the panel, a small peg is used. It is removable without the need for tools (it uses a snap fit design). 

The two new STLs are oriented in their ideal print direction.

I can provide a Fusion360 file for the source if that's worthy of a contribution.

printed prototype

3d model face view

3d model back view

### CODE REVIEW

**Comment by aderusha** [comment]
That looks great!  Ready to merge this when you are, I bet some folks would love this.

**Comment by jensbodal** [comment]
Related to your comment regarding a rubber mallet in your README update, I found a $6 jeweler's hammer from Harbor Freight works great for final assembly. I'll use vice grip pliers initially to try and set a pin, then just set on a flat surface to hammer down using the nylon side. Also works well to lock in pieces on the board itself if it's a really tight fit. 

https://www.harborfreight.com/hand-tools/hammers-pry-bars/mallets/jewelry/double-sided-mallet-with-wooden-handle-98285.html

File: Quickhooks/README.md [hunk 1]
```diff
@@ -8,4 +8,4 @@ Quickhooks are fast-printing, two part hook/hangar solutions designed for quick
 
 ## Printing notes
 
-Quickhooks should be suitable for fast printing, set your slicer up for maximum perimeters to ensure a strong printed result.  The retainer is press fit into the quickhook body.  I use a clamp to apply a steady force to press the retainer into place.
+Quickhooks should be suitable for fast printing, set your slicer up for maximum perimeters to ensure a strong printed result.  The retainer is press fit into the quickhook body. I use a clamp to apply a steady force to press the retainer into place. A rubber mallet is another good way to press fit together a quickhook and a retainer.
```
Comment (jensbodal) [inline Quickhooks/README.md:11]: I've also found a $6 jeweler's hammer from Harbor Freight works great for final assembly. I'll use vice grip pliers initially to try and set a pin, then just set on a flat surface to hammer down using the nylon side. Also works well to lock in pieces on the board itself if it's a really tight fit.

https://www.harborfreight.com/hand-tools/hammers-pry-bars/mallets/jewelry/double-sided-mallet-with-wooden-handle-98285.html

**Comment by yoannchaudet** [comment]
@aderusha Ready to merge! Added the source file for posteriority.

**Pull request merged**